src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -41,7 +41,13 @@
             <argument type="service" id="api_platform.router" />
             <argument type="service" id="api_platform.property_accessor" />
         </service>
-
+        <service id="api_platform.documentation" class="ApiPlatform\Core\Documentation\Documentation" public="false">
+            <argument>%api_platform.title%</argument>
```

Comment:
So, those parameters should be injected in the action. The action will pass them to the VO constructor.